public/stylesheets/configStyle.css

```diff
@@ -0,0 +1,64 @@
+.main-screen {
+    position: relative;
+    z-index: 1;
+    width: 100%;
+    margin: 10px auto;
+    padding: 15px;
+    min-height: 300px;
+    height: auto;
+}
+
+.mini-box {
+    position: relative;
+    width: 30%;
+    background: #fff;
```

Comment:
Maken wij gebruik van bootstrap? Zo ja, dan kan je daar ook standaard classes van pakken om styling toe te passen. Er is bijvoorbeeld een class om een witte achtergrond te zetten.

Reply:
tbh geen idee of we het gebruiken maar het is niet alsof het echt iets gaat veranderen

Reply:
Dit hebben we inderdaad nog niet besproken. Wellicht iets wat morgen kort overlegd kan worden? @Stupers